addons/edi_notifier/models/edi_document.py

```diff
@@ -0,0 +1,34 @@
+import logging
+
+from odoo import fields, api, models
+
+_logger = logging.getLogger(__name__)
+
+
+class EdiDocumentType(models.Model):
+    _inherit = "edi.document.type"
+
+    notifier_ids = fields.Many2many(
+        "edi.notifier", column1="doc_id", column2="notifier_id", string="Notifiers"
+    )
+
+    x_last_checked_not_received = fields.Datetime("Last not recived check")
```

Comment:
Typo: `recived`

Reply:
Also do we want to change it to just last_checked ?

Reply:
We might want to have multiple checks if we did we would need another field